webpack.config.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        exclude: /node_modules/,
```

Comment:
Generally, your webpack loaders only get applied to your source files, not dependencies, since they're typically shipped as ES3/5.